src/fileView.js

```diff
@@ -33,10 +33,9 @@ function renderPDFPreview(file) {
             <i class="fas fa-spinner fa-pulse"></i>
             <span id="loading-progress">Loading PDF...</span>
           </div>
-          <script src="https://cdn.jsdelivr.net/gh/pipwerks/PDFObject/pdfobject.min.js"></script>
           <script>
-          const loadingLabel = document.querySelector('.loading-label')
-          const loadingProgress = document.querySelector('#loading-progress')
+          loadingLabel = document.querySelector('.loading-label')
+          loadingProgress = document.querySelector('#loading-progress')
```

Comment:
Why are we deleting the `const` declarations?

Reply:
It's causing exception. Open 1.pdf, then return to folder, than 2.pdf could reproduce.